backend/controllers/userController.js

```diff
@@ -17,162 +17,164 @@ const authUser = asyncHandler(async (req, res) => {
       email: user.email,
       isAdmin: user.isAdmin,
       token: generateToken(user._id),
-    })
+    });
   } else {
-    res.status(401)
-    throw new Error('Invalid email or password')
+    res.status(401);
+    throw new Error('Invalid email or password');
   }
-})
+});
 
 // @desc    Register a new user
 // @route   POST /api/users
 // @access  Public
 const registerUser = asyncHandler(async (req, res) => {
-  const { name, email, password } = req.body
+  const { name, email, password, isAdmin } = req.body;
```

Comment:
Functionality change: registerUser takes uses isAdmin to set admin status since isAdmin is set by Auth0.